### ISSUE TITLE
Characters breaking/unable to interact with game after clothing touched by others.

### ISSUE THREAD

**Issue by lilahnovi** [tgstation/tgstation#80501]
## Issue Summary

Characters seem to be breaking, unable to look around (e.g. clicking a faraway tile to change direction), and unable to use their hands. They can still walk and talk. This seems to happen after someone:
- Puts a clothing object on someone, like a suit.
- Adjusts clothing to be worn casual.
- Take clothing off someone.

Supposedly, some fixes are:
- Ahelp and get deleted and respawn in.
- Cultist of Pupcup states: "while your getting cuffed and uncuffed, spamclick your bag." (I haven't tried this fix, but someone else said it worked for them.)
- MentalDecayed states: "open task manager, look for the program 'file explorer' and choose to restart task. It's because windows 11 looooooves to open up invisible windows on top of ss13." (For the record here, I am running Windows 10, and still had this issue, but I have not tried this fix.)

## Round ID:
[220904](https://scrubby.melonmesa.com/round/220904)

However, this issue has been happening at least since yesterday (12/20). I noticed it when I started playing medical for the first time. I broke a lot of people taking off their clothing. I will comment with more specific circumstances (what clothing, exactly what was I doing, etc) on this later if it happens again.

**Comment by Mothblocks** [comment]
Related #80492 ?

**Comment by MrMelbert** [comment]
Maybe caused by #79498 , please include test merges

**Comment by lilahnovi** [comment]
> Related #80492 ?

It might be. It sounds similar. However, my issue was never that I was being affected -- it was the people around me that had their clothing futzed with that ended up borking.

I got affected once during a round (if you remember Mothblocks, the one where you healed me on Delta station when Effie was chef - my clothes were removed here. Then I was converted to rev on the table. I had to go into a corner to get ahelp deleted and re-spawned.)
